acme/acme/challenges.py

```diff
@@ -436,10 +437,14 @@ def gen_cert(self, domain: str, key: Optional[crypto.PKey] = None, bits: int = 2
             key.generate_key(crypto.TYPE_RSA, bits)
 
         der_value = b"DER:" + codecs.encode(self.h, 'hex')
```

Comment:
This value wont' work with a cryptography `x509.UnrecognizedExtension`, `DER:blah blah` is an OpenSSL-ism. It looks like `self.h` is already some bytes, so you can just pass that directly, assuming that's the entire contents of the extension

Reply:
good catch, thanks!